Opret togt

JRJ86 opened the pull request.

JRJ86 committed:
drop down skibs liste oprettet, i OpretTogt_frag, men ikke stylet

JRJ86 committed:
Opret togt layout is ongoing

JRJ86 committed:
videre implementerer Togt klassen og oprettet Besaetning klassen, som skal bruges til OpretTogt_frag fragmentet

JRJ86 committed:
OpretTogt_frag førdig. minus besætnings fragmentet.

JRJ86 committed:
Recycleview for TogtOversigten er done

JRJ86 committed:
TogtOversigt list elements are clickable
comments missing

JRJ86 committed:
made a floating button in the togt list to create a new "Togt"

JRJ86 committed:
recyclerview commencing

JRJ86 committed:
comments

JRJ86 committed:
comments

JRJ86 committed:
edit and delete button created in togt recycleview

JRJ86 committed:
Merge branch 'development' into opretTogt

# Conflicts:
#	.idea/misc.xml
#	app/src/main/java/com/example/skibslogapp/Main_akt.java
#	app/src/main/java/com/example/skibslogapp/model/Togt.java
#	app/src/main/java/com/example/skibslogapp/view/TogtOversigt_frag.java
#	build.gradle

JRJ86 committed:
bug fixings

JRJ86 committed:
bug fixings

JRJ86 committed:
Togt model, and OpretTogt_frag have been edited abit to acomodate new informations

JRJ86 committed:
Erase from the Togt list done

JRJ86 committed:
bug fixing

JRJ86 committed:
Merge remote-tracking branch 'origin/db_info_update' into opretTogt

# Conflicts:
#	app/src/main/java/com/example/skibslogapp/model/Togt.java
#	app/src/main/java/com/example/skibslogapp/view/TogtOversigt_frag.java

JRJ86 committed:
when you create a "Togt", you put it in the DB, and not SharedPreferences

JRJ86 committed:
comments in TogtListAdapter, TogtOversigt_frag, OpretTogt_frag and Main_akt

JRJ86 committed:
main act comments

JRJ86 committed:
Merge remote-tracking branch 'origin/db_info_update' into opretTogt

# Conflicts:
#	app/src/main/java/com/example/skibslogapp/datalayer/local/TogtDAO.java

JRJ86 committed:
Merge remote-tracking branch 'origin/development' into opretTogt

# Conflicts:
#	app/src/main/java/com/example/skibslogapp/Main_akt.java

JRJ86 committed:
delete "Togt" from DB

AndreasBGJensen committed:
Rearange: TogtOversigt_fraq

AndreasBGJensen committed:
Merge remote-tracking branch 'origin/development' into opretTogt

AndreasBGJensen committed:
Merge Developer into OpretTogt

AndreasBGJensen commented:
Ser godt ud!
Developer er blevet Merget ind
Har lige er par spørgsmål / kommentarer- Beklager, hvis disse ting er blevet diskuteret tidligere

Spørgsmål:
fragment_opret_log: 
- HVorfor er anvendes der ScrollView i fragmentet?

Opret_Togt-fraq
- Skal skibene have en navne? - Afventer vi evt, hvilke navne vi skal anvende?

Togt_oversigt:
- Er feltet, hvor der står "GlobalTogt" en overskrift? Hvor bliver den sat? Hvis det er en overskrift syntes jeg at den skla have samme styling som "Opret Togt" i OpretTogt_fraq (="@+id/opretTogtOverskrift")


Generelt: Måske vi skal flytte changeFragment ud i utility mappen da vi bruger den i både Opret_Togt_fraq og Togt_oversigt_fraq og mange andre steder.

AndreasBGJensen reviewed (commented):
Ser godt ud!
Developer er blevet Merget ind
Har lige er par spørgsmål / kommentarer- Beklager, hvis disse ting er blevet diskuteret tidligere

Spørgsmål:
fragment_opret_log:

    HVorfor er anvendes der ScrollView i fragmentet?

Opret_Togt-fraq

    Skal skibene have en navne? - Afventer vi evt, hvilke navne vi skal anvende?

Togt_oversigt:

    Er feltet, hvor der står "GlobalTogt" en overskrift? Hvor bliver den sat? Hvis det er en overskrift syntes jeg at den skla have samme styling som "Opret Togt" i OpretTogt_fraq (="@+id/opretTogtOverskrift")

Generelt: Måske vi skal flytte changeFragment ud i utility mappen da vi bruger den i både Opret_Togt_fraq og Togt_oversigt_fraq og mange andre steder.

JRJ86 committed:
known ship names implemented in the ship list

JRJ86 committed:
Merge branch 'development' into opretTogt

# Conflicts:
#	app/src/main/java/com/example/skibslogapp/Main_akt.java

JRJ86 committed:
bug fixing. You can change to OpretTogt_frag from side menu now

AndreasBGJensen committed:
Merge remote-tracking branch 'origin/development' into opretTogt

# Conflicts:
#	app/build.gradle
#	app/src/main/java/com/example/skibslogapp/Main_akt.java
#	app/src/main/res/values/strings.xml

AndreasBGJensen committed:
Add OpretLog_fraq navigation from side menu:
Before, when navigating the app was chrashing with a null pointer exception.

This still makes complications when navigating to OpretLog_fraq from SideMenu because we need the Open/Close button from the PostOversigt fragment. So for this moment one can only navigate away from OpretLog unsig the sidemenu.

maltebp committed:
Merge remote-tracking branch 'origin/development' into opretTogt

# Conflicts:
#	app/build.gradle
